ScopedStorage/app/src/main/java/com/samples/storage/mediastore/AddMediaViewModel.kt

```diff
@@ -36,11 +44,13 @@ class AddMediaViewModel(application: Application) : AndroidViewModel(application
     val isPermissionGranted: Boolean
         get() = canWriteInMediaStore(context)
 
+    private val httpClient by lazy { OkHttpClient() }
```

Comment:
Consider adding kdoc explaining why one would use lazy here vs just creating the OkHttpClient as a private member.